<!--
# Copyright (c) 2023, ANTOBOT LTD.
# All rights reserved.

# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # 
# # # Code Description:   
# Contact: soyoung.kim@antobot.ai
# # # #  # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-->

<launch>
  <arg name="exec_name" default="gps_rtk_mqtt.py" />
  <arg name="node_name" default="gps_rtk_mqtt" />
  <arg name="device_port" default="/dev/ttyTHS0" />
  <arg name="device_id" default="None" />
  <arg name="base_id" default="None" />

  <param name="gps_node_name" value="$(arg node_name)" />

  <node pkg="antobot_devices_gps" type="$(arg exec_name)" name="$(arg node_name)" output="screen">
    <param name="device_port" value="$(arg device_port)"/>
    <param name="device_id" value="$(arg device_id)" />
    <param name="base_id" value="$(arg base_id)" />
  </node>

</launch>
